src/bin/ion/commands/beta/generate/mod.rs

```diff
@@ -1,6 +1,7 @@
 mod context;
 mod generator;
 mod result;
+pub(crate) mod templates;
```

Comment:
Does this need to be `pub(crate)`?

Reply:
Not required. Changed this in the latest commit.